lib/slappy/event.rb

```diff
@@ -13,5 +13,19 @@ def initialize(data)
     def text
       @data['text'].to_s
     end
+
+    def channel
+      SlackAPI::Channel.find(id: @data['channel']) ||
+        SlackAPI::Group.find(id: @data['channel']) ||
+        SlackAPI::Direct.find(id: @data['channel'])
+    end
+
+    def user
+      SlackAPI::User.find(id: @data['user'])
+    end
+
+    def ts
+      Time.at(@data['ts'].to_f)
```

Comment:
Rails/TimeZone: Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.